app/src/main/java/com/jerboa/ui/components/community/CommunityActivity.kt

```diff
@@ -82,7 +83,7 @@ fun CommunityActivity(
     showVotingArrowsInListView: Boolean,
     useCustomTabs: Boolean,
     usePrivateTabs: Boolean,
-    blurNSFW: Boolean,
+    blurNSFW: Int,
```

Comment:
Why can't these use the BlurType enum?

Reply:
We don't do it either for other enums, see postActionMode below it